### PR TITLE
[ESIMD] Add a test exercising 2D workitem coordinates.

### PR DESCRIPTION
**Pull request by kbobrovs**
Signed-off-by: Konstantin S Bobrovsky <konstantin.s.bobrovsky@intel.com>

### CODE REVIEW

**Review by kychendev — DISMISSED** [review]

File: SYCL/ESIMD/vadd_2d_acc.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            int i = gid * 16 + lid;
+            unsigned int offset = i * VL * sizeof(float);
+            simd<float, VL> va = block_load<float, VL>(PA, offset);
+            simd<float, VL> vb = block_load<float, VL>(PB, offset);
+            simd<float, VL> vc = va + vb;
+            block_store(PC, offset, vc);
+          });
+    });
+    e.wait();
+  }
```
Comment (DenisBakhvalov) [inline SYCL/ESIMD/vadd_2d_acc.cpp:78]: Please add try/catch blocks.

**Review by DenisBakhvalov — CHANGES_REQUESTED** [review]

File: SYCL/ESIMD/vadd_2d_acc.cpp (hunk 1 above)
Comment (kbobrovs) [reply SYCL/ESIMD/vadd_2d_acc.cpp:78]: done

**Review by DenisBakhvalov — APPROVED** [review]

**Comment by vladimirlaz** [comment]
@kychendev is it OK to merge the PR? The tests are passing and there are no unresolved comments.

**Comment by kychendev** [comment]
> @kychendev is it OK to merge the PR? The tests are passing and there are no unresolved comments.

Yes, I didn't notice my last review was dismissed after last update. Approving now.

**Review by kychendev — APPROVED** [review]

**Pull request merged**